src/classes/issues-processor.ts

```diff
@@ -312,6 +353,151 @@ export class IssuesProcessor {
       }
     }
 
+    if (this.options.mlflow && !issue.isStale && !issue.milestone) {
```

Comment:
I think we should still remind maintainers / community members to interact with milestone issues, we just shouldn't mark stale / close milestone issues if they're inactive. Can we move the `!issue.milestone` check?